eslint.config.js

```diff
@@ -1,12 +1,63 @@
-import js from "@eslint/js";
+import eslintConfigESLint from "eslint-config-eslint";
+import eslintConfigPrettier from "eslint-config-prettier";
+
+const eslintPluginJSDoc = eslintConfigESLint.find(
+	config => config.plugins?.jsdoc,
+).plugins.jsdoc;
 
 export default [
-	js.configs.recommended,
 	{
-		ignores: ["**/tests/fixtures/**/*.*"],
+		ignores: ["**/tests/fixtures/", "**/dist/"],
+	},
+
+	...eslintConfigESLint,
+
+	// disable rules that are unnecessary or might conflict with Prettier
+	eslintConfigPrettier,
+
+	{
+		rules: {
+			// disable additional deprecated rules
+			"padding-line-between-statements": "off",
+			"spaced-comment": "off",
+
+			// disable rules we don't want to use from eslint-config-eslint
+			"no-undefined": "off",
```

Comment:
There were 34 problems reported by this rule. I assumed we don't want to use it in this repo.